common/djangoapps/util/views.py

```diff
@@ -434,6 +440,12 @@ def build_error_response(status_code, field, err_msg):
 
     success = False
     context = get_feedback_form_context(request)
+
+    #Update the tag info with 'enterprise_learner' if the user belongs to an enterprise customer.
+    enterprise_learner_data = enterprise_api.get_enterprise_learner_data(site=request.site, user=request.user)
```

Comment:
@brittneyexline what I did it yet to test this method locally. 
1- Created a enterprise learner and hit `enterprise_api.get_enterprise_learner_data(site=request.site, user=request.user)`, I got results. (working fine) 
2- Sign-in with non enterprise learner and hit `enterprise_api.get_enterprise_learner_data(site=request.site, user=request.user)`, I got emtpy list `[]` (working fine).